app/src/main/res/values/strings.xml

```diff
@@ -74,6 +74,8 @@
     <string name="browser_menu_your_library">Your Library</string>
     <!-- Browser menu toggle that requests a desktop site -->
     <string name="browser_menu_desktop_site">Desktop site</string>
+    <!-- Browser menu button that adds a top site to the home fragment -->
+    <string name="browser_menu_add_to_firefox_home">Add to Firefox home</string>
```

Comment:
This string seems like it could be easily confused with "Add to Home screen". Maybe "Pin/Add to Top sites" would be better?

Reply:
While I agree with your suggestion, this one is completely outside our control since it was decided by content strategy team.